infra/growth/src/util/money.js

```diff
@@ -5,24 +5,18 @@ class Money {
     if (values === null || values.length === 0) {
       return { amount: '0', currency }
     }
-    values = values.map(reward => reward.value)
-
-    const total = values.reduce((first, second) => {
-      if (first.currency !== second.currency)
-        throw new Error(
-          `At least two values have different currencies. ${first.currency} ${
-            second.currency
-          }`
-        )
-      return {
-        amount: BigNumber(first.amount).plus(BigNumber(second.amount)),
-        currency: first.currency
+    let total = BigNumber(0)
+    for (const value of values) {
+      console.log('ADDING ', value)
```

Comment:
this creeped in :)

Reply:
that was some light speed action removing this 🚀

Reply:
removed